lib/src/lib.rs

```diff
@@ -83,3 +89,18 @@ impl Files {
         }
     }
 }
+
+/// Iterates over the file names available for `Files` object.
+pub struct FileNames<'a> {
+    /// Our internal iterator.  We wrap this in a nice struct so our
+    /// caller doesn't need to know the details.
+    iter: phf::map::Keys<'a, &'static str, (Compression, &'static [u8])>,
```

Comment:
You don't actually need a lifetime parameter here. It theoretically allows to use `FileNames` in more situations, but all `Files` are going to be `'static`.

Leaving the lifetime parameter off decreases the API complexity by an amount so miniscule that nobody cares about it, but I just learned that you can remove it by replacing `'a` for `'static`, and the signature of `file_names` by `pub fn file_names(&'static self) -> FileNames`.